app/controllers/api/v1/locations_controller.rb

```diff
@@ -25,6 +25,12 @@ def create
     end
   end
 
+  def destroy
+    Location.find(params["id"]).destroy
+    locations = Location.all
+    render json: locations
+  end
+
```

Comment:
keep in mind that these endpoints are vulnerable to curl commands at the moment. Dont worry about it now, but you will want to come back around for user authorization to this endpoint as well as your update endpoint to ensure either an admin or the original creator are the only ones able to delete this spot

Reply:
but this is totally fine as is